package.json

```diff
@@ -62,7 +62,7 @@
     "react-helmet": "^5.2.0",
     "react-router-dom": "^4.3.1",
     "speculate": "^1.7.4",
-    "styled-components": "^3.3.3",
+    "styled-components": "3.1.6",
```

Comment:
Could you please explain here why you have pinned to this particular version of `styled-components`?

Reply:
Being done in another PR now, will met that get merged and it'll no longer be done here